src/main/java/chalmers/pimp/model/canvas/LayerUpdateEvent.java

```diff
@@ -138,9 +138,10 @@ public IReadOnlyLayer getSelectedLayer() {
   void setSelectedLayer(int index) {
     int i = 0;
     for (IReadOnlyLayer layer : layers) {
-      if (i++ == index) {
+      if (index == i) {
         selectedLayer = layer;
       }
+      i++;
```

Comment:
Would love some doc right here. Might be hard for someone less intelligent to understand this :neckbeard:

Reply:
Speak for yourself.

Reply:
Oof